File: bialydunajec-adminpanel-frontend/projects/bialydunajec-admin/src/app/camp-edition/service/rest/request/create-camp-edition.request.ts
export class CreateCampEditionRequest {
  campEditionId: number;
  campEditionStartDate: Date;
  campEditionEndDate: Date;
  campEditionPrice: number;


  constructor(campEditionId: number, campEditionStartDate: Date, campEditionEndDate: Date, campEditionPrice: number) {
    this.campEditionId = campEditionId;
    this.campEditionStartDate = campEditionStartDate;
    this.campEditionEndDate = campEditionEndDate;
    this.campEditionPrice = campEditionPrice;
  }
}

File: bialydunajec-camp-edition/presentation/src/main/kotlin/org/bialydunajec/campedition/presentation/rest/v1/request/CreateCampEditionRequest.kt
package org.bialydunajec.campedition.presentation.rest.v1.request

import java.time.LocalDate

data class CreateCampEditionRequest(
        val campEditionId: Int,
        val campEditionStartDate: LocalDate,
        val campEditionEndDate: LocalDate,
        val campEditionPrice: Double
)